ports/llvm-17/vcpkg.json

```diff
@@ -30,6 +30,19 @@
     "utils"
   ],
   "features": {
+    "liftingbits-llvm": {
```

Comment:
do we want to want to name this something like amp-llvm since our fork is for patch recompilation and not really that useful outside of amp